src/serde/timestamp.rs

```diff
@@ -0,0 +1,151 @@
+//! De/serialize [`OffsetDateTime`] from/to [Unix timestamps](https://en.wikipedia.org/wiki/Unix_time).
+//!
+//! Use this module in combination with [serde's with-annotation](https://serde.rs/field-attrs.html#with).
+//!
+//! Note that the timestamp represenatioon lacks [`UtcOffset`],
+//! which is being lost on serialization and assumed 0 on deserialization.
+//!
+//! # Examples
+//!
+//! ```
+//! # #[cfg(feature = "_serde_json")] {
+//! # use serde::{Deserialize, Serialize};
+//! # use _serde_json as serde_json;
+//! # use serde_json::json;
+//! use time::serde::timestamp;
+//! # use time::{date, time, OffsetDateTime};
+//!
+//! # #[derive(Debug, PartialEq)]
+//! #[derive(Deserialize, Serialize)]
+//! struct S {
+//!     #[serde(with = "timestamp")]
+//!     datetime: OffsetDateTime,
+//! }
+//!
+//! # fn test() -> Result<(), serde_json::Error> {
+//! let s = S {
+//!     datetime: date!(2019-01-01).midnight().assume_utc(),
+//! };
+//! let v = json!({ "datetime": 1_546_300_800 });
+//! assert_eq!(v, serde_json::to_value(&s)?);
+//! assert_eq!(s, serde_json::from_value(v)?);
+//! # Ok(())
+//! # }
+//! # test().unwrap();
+//! # }
+//! ```
+//!
+//! [`OffsetDateTime`]: crate::OffsetDateTime
+//! [`UtcOffset`]: crate::UtcOffset
```

Comment:
Docs are built on nightly (both for time-rs.github.io and docs.rs) with intra-crate links enabled, so this isn't necessary to specify.

Reply:
I removed the explicit link for `OffsetDateTime`, but the one for `UtcOffset` seems to be required, as it is not `use`d.